src/geometry/SphereSceneNode.cxx

```diff
@@ -214,10 +191,20 @@ void SphereLodSceneNode::kill()
 }
 
 
-SphereLodSceneNode::SphereLodSceneNode(const GLfloat pos[3], GLfloat _radius) :
-    SphereSceneNode(pos, _radius),
+SphereSceneNode::SphereSceneNode(const GLfloat pos[3], GLfloat _radius) :
```

Comment:
as an aside, (unscoped) names with leading underscores are reserved to the compiler. There is a style of writing that used a leading underscore for member variables; this is legal because they are scoped by the class. We should not be using parameters with leading underscores. A (single) trailing underscore is acceptable.

Reply:
Done